packages/workspace/src/command-line/affected.ts

```diff
@@ -47,7 +47,9 @@ export async function affected(
     switch (command) {
       case 'apps':
         const apps = filteredProjects
-          .filter((p) => p.type === ProjectType.app)
+          .filter(
+            (p) => p.type === ProjectType.app || p.type === ProjectType.e2e
```

Comment:
Since e2e aren't deployable apps I don't think we want to include them in the affected apps list.

It will break scenarios where it is used in CD to determine what apps to deploy.